src/components/common/BackButton/BackButton.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-btn class="back-button" icon @click="goBack">
```

Comment:
Follow the Template classes rules https://developers.adamant.im/template-classes.html